### PR TITLE
# 02 Automating Tasks

### PR DESCRIPTION
**Pull request by TheNachi**
#### What does this PR do?
 - Merges changes in bayo-izanami-mugorji to bayo-izanami-mugorji-output. Output for Checkpoint 1
 #### Description of Task to be completed?
 - A README.md file with a link to a YouTube video of a screen-cast showing gulp watching my HTML/CSS/JS files for changes
 #### How should this be manually tested?
 - By visiting the you tube link and watching the video
 #### Any background context you want to provide?
 - n/a
 #### What are the relevant pivotal tracker stories?
 - n/a
 #### Video (if appropriate)
 - <https://www.youtube.com/watch?v=FUhsM_Sq-6I>
 #### Questions:
- n/a

### CODE REVIEW

**Comment by andela-cofor** [comment]
Detailed looks good to me...

**Comment by victor-enogwe** [comment]
looks good.

**Comment by TheNachi** [comment]
Thanks @andela-cofor @andela-venogwe

File: 06-building-a-program-start-to-finish/setting-up-your-repository/Output/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Setting up repository
+
+
+## Output
+- **Used**: GitHub
+- **Deliverable**: A repository for proposed project and at least one Pull Request
+
+
+## Link to my GitHub Repo
+<https://github.com/andela-mugorji/Inverted-Index>
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-your-repository/Output/README.md:10]: Please ensure you fill the missing sections on the ReadMe as you move along.

Comment (TheNachi) [reply 06-building-a-program-start-to-finish/setting-up-your-repository/Output/README.md:10]: Ok, will do so. Thanks

**Pull request merged**